types/react-native/test/index.tsx

```diff
@@ -909,3 +910,18 @@ const PermissionsAndroidTest = () => {
         }
     })
 }
+
+// Platform
+const PlatformTest = () => {
+    if (Platform.OS === 'ios') {
+        if (!Platform.isPad) {
+            return 32;
+        } else {
+            return 44;
+        }
+    } else if (Platform.OS === 'android') {
+        return 56;
+    } else {
+        return 64;
+    }
+};
```

Comment:
Can you update this test to access platform specific properties inside the android and else branches too?

Reply:
Added